assets/js/models/service_record.ts

```diff
@@ -0,0 +1,6 @@
+export interface ServiceRecord {
+  route: string;
+  place_id: string;
```

Comment:
Is it consistent within Screenplay that we use "place" to refer to a parent station, or can it mean something else? Here it seems to exactly mean "station" (stop with location type 1).

Reply:
The full places list includes bus stops as well. But I guess this could be `stop_id` or `station_id`.

Reply:
Oh, gotcha, so it's "stop with no parent station" (a.k.a. either a parent station or standalone stop... it's annoying there's no more concise terminology for that). But yeah I think it's fine as `place_id`.